api/v2/serializers/fields/status_type.py

```diff
@@ -8,8 +8,7 @@ class StatusTypeRelatedField(serializers.RelatedField):
     def get_queryset(self):
         return StatusType.objects.all()
 
-    def to_representation(self, value):
-        status_type = StatusType.objects.get(pk=value.pk)
+    def to_representation(self, status_type):
         serializer = StatusTypeSummarySerializer(
```

Comment:
The line on 13 is notorious. The resource request viewset doesnt use this field anymore. See the comment above.